src/SIL.XForge.Scripture/ClientApp/src/material-styles.scss

```diff
@@ -51,33 +51,81 @@ $mat-sf-orange: (
   )
 );
 
+$mat-sf-error: (
+  100: $errorColor,
+  500: $errorColor,
+  700: $errorColor,
+  contrast: (
+    100: white,
+    500: white,
+    700: white
+  )
+);
+
 // Styles for angular material components. Refer to https://material.angular.io/guide/theming
-$matGrey: mat.define-palette($mat-sf-grey, 800);
-$matGreenLight: mat.define-palette($mat-sf-green, 500, 200);
+$matGreyPalette: mat.define-palette($mat-sf-grey, 800);
+$matGreenLightPalette: mat.define-palette($mat-sf-green, 500, 200);
+$matErrorPalette: mat.define-palette($mat-sf-error, 500);
 
 $material-app-theme: mat.define-light-theme(
   (
     color: (
-      primary: $matGrey,
-      accent: $matGreenLight
+      primary: $matGreyPalette,
+      accent: $matGreenLightPalette,
+      warn: $matErrorPalette
     )
   )
 );
 
+// A Material input uses the accent color for the asterisk, but light green is an odd color to use there.
+// This theme uses the error color for accent and can be applied only where needed.
+$material-theme-accent-error: mat.define-light-theme(
```

Comment:
Here's what I was trying to avoid (note the barely visible light green asterisk):
![form_field_no_distinct_theme](https://github.com/sillsdev/web-xforge/assets/6140710/0a8f17c1-914c-4b21-8a41-cc2e6917d18d)